code/controllers/voting.dm

```diff
@@ -1,9 +1,15 @@
 var/datum/controller/vote/vote = new()
 
-var/global/list/round_voters = list() //Keeps track of the individuals voting for a given round, for use in forcedrafting.
+var/global/list/round_voters = list() // Keeps track of the individuals voting for a given round, for use in forcedrafting.
 
-datum/controller/vote
-	var/initiator = null
+#define VOTE_RESTART "restart"
+#define VOTE_GAMEMODE "gamemode"
+#define VOTE_CREW_TRANSFER "crew_transfer"
+#define VOTE_ADD_ANTAGONIST "add_antagonist"
+#define VOTE_CUSTOM "custom"
+

```

Comment:
They replace the strings. Previously, there were repeated uses of ``"restart"`` as a string. Which is prone to typos, because typing ``"rsetart"`` will not cause any compilation errors, but ``VOTE_RSETART`` will. This is a pretty common practice.